### PR TITLE
[4.1.x][6628] Add a way to preview audio and .srt files via UI

### PR DESCRIPTION
**Pull request by jvega190**
Add a way to preview audio and .srt files via UI https://github.com/craftercms/craftercms/issues/6628
Add support for audio icons in Sidebar https://github.com/craftercms/craftercms/issues/6677
Add subtitle files and audio files to the authoring index https://github.com/craftercms/craftercms/issues/6536

### CODE REVIEW

**Comment by jvega190** [comment]
@rart please note that even though https://github.com/craftercms/craftercms/issues/6628 doesn't include 4.1.x, #6677 has a request for the same thing #6628 requests (https://github.com/craftercms/craftercms/issues/6677#issuecomment-2045782069). So I included the PreviewDialog support

File: ui/app/src/components/MediaCard/MediaCard.tsx [hunk 1]
```diff
@@ -106,7 +106,11 @@ function MediaCard(props: MediaCardProps) {
     compact = false
   } = props;
   // endregion
-  const { name, path, type } = item;
+  let { name, path, type } = item;
+  // TODO: search item type comes as 'Other' in audio files, it needs to be updated to 'Audio'
```
Comment (rart) [inline ui/app/src/components/MediaCard/MediaCard.tsx]: On the back? Is there a ticket? Start the discussion.

**Review by rart — APPROVED** [review]

File: ui/app/src/components/MediaCard/MediaCard.tsx (hunk 1 above)
Comment (rart) [reply ui/app/src/components/MediaCard/MediaCard.tsx:110]: Is there anything to update here or is this just back? If just back, remove Todo and follow up on discussion

Comment (rart) [inline ui/app/src/components/MediaCard/MediaCard.tsx]: Is there anything to update here after that change, or is this just backend stuff? If just back, remove Todo and follow up on discussion

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/MediaCard/MediaCard.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/MediaCard/MediaCard.tsx]: Only backend, removing

**Review by rart — APPROVED** [review]

**Pull request merged**